tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,29 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+
+class BugzillaLite(Base):
+
+    name = 'Bzlite'
+    _given_username = ('css selector', "#login input[type='email']")
+    _given_password = ('css selector', "#login input[type='password']")
+    _button_login = ('css selector', "#login input[type='submit']")
+    # home frame
+    home_frame = marionette.find_element('css selector', 'div.homescreen iframe')
+    marionette.switch_to_frame(home_frame)
+    #launch
+    def launch (self):
+        BugzLite_icon = marionette.find_element('xpath', "//div[@class='icon']//span[contains(text(),'Bugzilla Lite')]")
+        BugzLite_icon.tap()
+    
+    #Switch context
+    self.marionette.switch_to_frame()
+    bugzillaLite_frame = marionette.find_element('css selector', "iframe[data-url*='bzlite.com']")
+    self.marionette.switch_to_frame(bugzillaLite_frame)
```

Comment:
Now that we use `GaiaTestCase`, we have a handy function called `self.apps.switch_to_displayed_app()`